src/main/java/berlin/yuna/nano/helper/NanoUtils.java

```diff
@@ -25,6 +29,11 @@
 public class NanoUtils {
 
     public static final String LINE_SEPARATOR = System.lineSeparator();
+    protected static Random random;
+    public static final String[][] NANO_NAMES = {
```

Comment:
Should we make it protected/private as it only use inside class, can be change by user

Reply:
Nice catch. But I think this `immutable` field, can be public, since `NanoUtils` is `static`. You can't create an instance and can't add getters and setters. The user would not get access easily when making it `protected`.
Regarding `private`, nothing should be ever `private` as this would limit the user from using or overriding parts if neccessary.